notebooks/geo/DEM_MapVisualization.ipynb

```diff
@@ -0,0 +1,215 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    pds4_tools.view(xml_urls[0][0])
</pre><p>When I run the pds4_tools the second time it takes forever and then python crashes.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-PDS/search-api-notebook/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>